Attendance_app.py

```diff
@@ -192,9 +196,21 @@ def __init__(self, parent, controller):
             font=("Courier", text_fontsize),
         )
         self.lb_class.place(x=170 * scale_factor, y=155 * scale_factor)
-        self.tv_class = tk.Entry(self, width=entry_width)
-        self.tv_class.focus()
-        self.tv_class.place(x=340 * scale_factor, y=160 * scale_factor)
+
+        # Class Codes Autocompletion Stuff
+        self.filled = False
+
+        self.classCodes =class_codes

```

Comment:
⚠️ `E225 missing whitespace around operator.`
Forgot to apply code formatter?

⚠️ `camelCase usage for variable name`
Please use snake_case for variable names.